sdk/textanalytics/Azure.AI.TextAnalytics/src/autorest.md

```diff
@@ -11,6 +11,23 @@ input-file:
 generation1-convenience-client: true
 ```
 
+### Modify operationId names
+
+``` yaml
+directive:
```

Comment:
Nice!   I didn't know that you could do this.

Reply:
Autorest has a lot of magic. If only we knew it all :P